src/index.js

```diff
@@ -281,17 +281,44 @@ export default function (babel) {
     return (typeof node.skinny === "boolean");
   }
 
+  function getCompletionRecords(path) {
+    let paths = [];
+
+    const add = function add(_path) {
+      if (_path) paths = paths.concat(getCompletionRecords(_path));
+    };
+
+    if (path.isIfStatement()) {
+      add(path.get("consequent"));
+      add(path.get("alternate"));
+    } else if (path.isDoExpression() || path.isFor() || path.isWhile()) {
```

Comment:
(we'll want to pass call it with allowLoop `false` for implicitReturns and `true` for comprehensions